contributors.md

```diff
@@ -14,6 +14,7 @@ To check the project builds succesfully, run the following command:
 ```bash
 bazel build ...
 ```
+_Note: If you are running Bazel on a **Mac**, `bazel build...` may fail with a `Too many open files in system` error. This is [due to a limitation](https://github.com/bazelbuild/rules_closure/issues/218) on the default maximum open file descriptors. For OS Sierra you can increase this limit with this  command: `sudo sysctl -w kern.maxfiles=16384`._
```

Comment:
nit: not sure if that github issue is super helpful, this might give a little more detail: https://github.com/angular/angular-bazel-example/issues/178

Reply:
Done